metrics.go

```diff
@@ -154,12 +155,26 @@ func (se *statsExporter) metricToMpbTs(ctx context.Context, metric *metricdata.M
 			// TODO: (@rghetia) perhaps log this error from labels extraction, if non-nil.
 			continue
 		}
+
+		var rsc *monitoredrespb.MonitoredResource
+		var mr monitoredresource.Interface
+		if get := se.o.ResourceByDescriptor; get != nil {
```

Comment:
Nit: why not just `se.o.ResourceByDescriptor != nil`?

Reply:
done.